ExampleApp Android/src/main/java/com/rootstrap/flowforms/example/SignUpFormActivity.kt

```diff
@@ -76,10 +79,15 @@ class SignUpFormActivity : AppCompatActivity() {
     }
 
     private fun onEmailStatusChange(status: FieldStatus) {
+        binding.emailAvailableText.visibility = View.GONE
```

Comment:
Maybe we could use `binding.apply { ... }` in order to simplify the code

Reply:
Yes, it fits very well here. Thanks!